src/components/Navigation/_variables.scss

```diff
@@ -47,6 +47,21 @@ $nav-animation-variables: (
     line-height: $item-line-height-large;
     padding-left: spacing() - spacing(extra-tight);
     padding-right: spacing(extra-tight);
+
+    @include breakpoint-before(nav-min-window-corrected()) {
+      // stylelint-disable-next-line selector-max-combinators
+      .Icon {
+        margin-top: nav(mobile-spacing);
+        margin-bottom: nav(mobile-spacing);
+      }
+
+      // stylelint-disable-next-line selector-max-combinators
+      .Text {
+        margin-top: nav(mobile-spacing);
+        margin-bottom: nav(mobile-spacing);
+        font-size: rem(15px);
```

Comment:
Looks like we're getting a height of 41px on nav items. If we add a rule for line-height here, setting it to `rem(19px)` we'll get a height of 40px